helper-cli/src/main/kotlin/common/Utils.kt

```diff
@@ -119,6 +121,14 @@ internal fun findRepositories(directory: File): Map<String, VcsInfo> {
     return ortResult.repository.nestedRepositories
 }
 
+/**
+ * Build the file for the split curations.
+ */
+internal fun getSplitCurationFile(parent: File, packageId: Identifier, fileExtension: String) =
+    parent.resolve(packageId.type.encodeOrUnknown())
+        .resolve(packageId.namespace.fileSystemEncode())
+        .resolve("${packageId.name.encodeOrUnknown()}.$fileExtension")
```

Comment:
Note that file.resolve("a/b/c") also works.

Reply:
While it works, does it offer any benefits?

Reply:
IMO chaining the `resolve()` calls is more clear than adding everything into `"${...}/${...}"`